server/server/controllers/customer.js

```diff
@@ -30,4 +30,19 @@ async function getUsers(req, res) {
   res.json(users);
 }
 
-module.exports = { signup, getUsers };
+async function signin(req, res) {
+  const { email, password } = req.body;
+  try {
+    const hashedPassword = await bcrypt.hash(password, 10);
```

Comment:
Suggestion to code: Customer's password should have been already hashed during sign up, so during sign in process, bcrypt.compare should be used between the password saved in the database and the password sent in the request body

Reply:
thanks for pointing that out